lockbox-ios/Presenter/FxAPresenter.swift

```diff
@@ -39,20 +56,50 @@ class FxAPresenter {
         self.accountStore = accountStore
     }
 
+    @available(iOS 12, *)
```

Comment:
Is there a corresponding init that's available not on iOS 12?

Reply:
yep ln50 has a init that is for other versions.